container/incremental_load.sh.tpl

```diff
@@ -239,8 +239,9 @@ function read_variables() {
 %{tag_statements}
 
 # An optional "docker run" statement for invoking a loaded container.
-# This is not executed if the single argument --norun is passed.
-if [ "a$*" != "a--norun" ]; then
+# This is not executed if the single argument --norun is passed or
+# no run_statements are generated (in which case, 'run' is 'False').
+if [[ "a$*" != "a--norun" && "%{run}" == "True" ]]; then
```

Comment:
You can probably reuse run_statements and check if that's not an empty string here.

Reply:
I was hesitant to do that -- `run_statements` can be modified by the consumer's `docker_run_flags` attribute, and this might contain double quotes.